src/components/About.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
-import { Wrapper } from './About.css';
-import { color } from '../styles/theme';
-import { Text } from '../styles/GlobalStyle.css';
+import { Wrapper, MockUp, Info, AboutTitle, AboutText, AboutButton } from './About.css';
```

Comment:
Mam wrażenie, że importowanie ostylowanych komponentów z osobnego pliku nie zawsze ma sens. Na pewno warto importować takie ostylowanie komponenty używane w wielu miejściach (jak Button), ale jeśli taki komponent klasowy używany jest tylko w jednym komponencie, to łatwiej się połapać, jeśli jest zdefiniowany w tym samym pliku, co komponent (i bez export)